transformer/test_locust.py

```diff
@@ -52,42 +52,6 @@ class LocustForScenarioGroup(HttpLocust):
         assert expected.strip() == script.strip()
 
 
-def test_generates_passed_global_code_blocks():
```

Comment:
Is there an equivalent test somewhere?

Reply:
That test was removed by accident (it started failing because of a change that I ultimately removed). Thank you for your eagle eye! :+1: